source/WorkingPractices/TestSuites/multi-repo_testing.rst

```diff
@@ -0,0 +1,104 @@
+.. _multi-repo_testing:
+
+Multi-Repository Testing
+========================
+
+Multi-repository changes are expected to pass the regression tests for all the
+repositories involved. To carry out the tests involved in a linked ticket it can
+be helpful to refer to the semi-concentric circles above; layering the testing
```

Comment:
semi concentric circle picture isn't on this page so this probably wants to be "semi-concentric circles here" where here is a link to the multirepo page.